permissions/Sensors.permission

```diff
@@ -5,3 +5,7 @@
 # x-sailjail-description = Sensors
 # x-sailjail-translation-key-long-description =
 # x-sailjail-long-description =
+
+dbus-system.talk com.nokia.SensorService
+
+whitelist /var/run/
```

Comment:
If this is for /var/run/sensord.sock access, there ought to be no reason to whitelist the containing directory, or?

Reply:
Whitelisting just the socket is not possible because of limitations in firejail.

Reply:
The socket was moved /run/sensord.sock which is automatically allowed and this line was removed.